src/pages/dataset/[accession_id].astro

```diff
@@ -12,42 +14,54 @@ export function getStaticPaths() {
   });
 }
 
+const table_of_contents = {
+  "Study Information": "study",
+  "Viewable Images": "viewable_images",
+  "All Images": "all_images",
```

Comment:
the all images link doesn't work, are you planning to add the all images table in the future?

Reply:
Yes, I don't think we have a table for this yet, but i figured i would leave in the heading for now since the annotation pages have all the sections they will have even if they're currently missing.